crates/compiler/build/src/program.rs

```diff
@@ -326,6 +336,9 @@ pub fn gen_from_mono_module_llvm(
         let mut app_bc_file = PathBuf::from(roc_file_path);
         app_bc_file.set_extension("bc");
 
+        let mut app_o_file = PathBuf::from(roc_file_path);
+        app_o_file.set_extension("o");
+
```

Comment:
here we need a temporary `.o` file for the llvm tooling to use. This is a debug path though, so I don't care that this is a bit inefficient.